.rubocop.yml

```diff
@@ -86,6 +86,11 @@ Metrics/CyclomaticComplexity:
 Metrics/ParameterLists:
   CountKeywordArgs: false
 
+# Reason: Prefer seeing a variable name
+# https://docs.rubocop.org/rubocop/cops_naming.html#namingblockforwarding
+Naming/BlockForwarding:
+  EnforcedStyle: explicit
```

Comment:
The change here is for something like

```ruby
def something(&block)
  other(&block)
end
```

The new default cop preference is use anonymous forwarding:

```ruby
def something(&)
  other(&)
end
```

I set this to `explicit` for now. I could also see accepting this change (I think the compactness would be nice but it might be jarring if not familiar) and/or throwing it into the _todo.